packages/roosterjs-editor-dom/lib/style/removeGlobalCssStyle.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Remove a css rule style from a style sheet
+ * @param doc The document object
+ * @param styleId the ID of the style tag
+ */
+
+export default function removeGlobalCssStyle(doc: Document, styleId: string) {
+    const styleTag = doc.getElementById(styleId) as HTMLStyleElement;
+    if (styleTag) {
+        doc.head.removeChild(styleTag);
```

Comment:
Use `styleTag.parentNode?.removeChild(styleTag)` instead

Although accoding to our code, style tag should always be under HEAD. But if we changed that behavior, there will be no build time error or test case to remind us change here. So better make it more robust